src/databricks/labs/remorph/cli.py

```diff
@@ -121,7 +120,7 @@ def transpile(
     for error in errors:
         print(str(error))
 
-    print(json.dumps(status))
+    logger.info(f"Finished transpilation: {status}")
```

Comment:
this is the part of table template in `labs.yml` this should have broken that template.